src/test/server/game/Miscellaneous/FormulasTest.cpp

```diff
@@ -1,13 +1,71 @@
 #include "gtest/gtest.h"
 #include "Formulas.h"
+#include "SharedDefines.h"
+
+using namespace acore::Honor;
+using namespace acore::XP;
```

Comment:
Should there be a semicolon at the end here?
On github diff they are red o.o

Reply:
what do you mean? I do see a semicolumn :D

Reply:
```cpp
using namespace acore::Honor;
using namespace acore::XP;
```

Reply:
at the end

Reply:
well yes, otherwise it would be

```
FormulasTest.cpp:5:29: error: expected ';' after namespace name
```

Reply:
weird that it shows as red